doc/rtd/reference/datasources/aeza.rst

```diff
@@ -0,0 +1,36 @@
+.. _datasource_aeza:
+
+Aeza
+****
+
+The `Aeza`_ datasource consumes the content served from Aeza's metadata
+service. This metadata service serves information about the running VPS
+via http at ``77.221.156.49``.
+
+Configuration
+=============
+
+Aeza's datasource can be configured as follows: ::
+
+  datasource:
+    Aeza:
+      metadata_url: "http://77.221.156.49/v1/cloudinit/{id}/meta-data"
+      userdata_url: "http://77.221.156.49/v1/cloudinit/{id}/user-data"
+      vendordata_url: "http://77.221.156.49/v1/cloudinit/{id}/vendor-data"
+
+* ``metadata_url``
+
+  Specifies the URL to retrieve the VPS meta-data. (optional)
```

Comment:
If the intent of `(optional)` is to say you don't have to provide this datasource config key, let's also provide the `Default: http://77.221.156.49/v1/cloudinit/{id}/`

Reply:
Fixed in 4e8b32dfd7ce0751ca0781a6894600e2e70f1a2a